pkgs/by-name/gl/glycin-loaders/package.nix

```diff
@@ -47,14 +50,23 @@ stdenv.mkDerivation (finalAttrs: {
   ];
 
   buildInputs = [
+    gobject-introspection
```

Comment:
Why do we need g-i when we do not build the C library (so there is nothing to bind to).

Reply:
part of the new meson.build but just noticed it is only for testing.

```
if get_option('tests')
  dependency('gio-2.0', version: gio_req)
```

https://gitlab.gnome.org/sophie-h/glycin/-/compare/1.0.1...1.1.1?page=4#e7efb14c7d4883f4146c2a32ae45f984ca9c1ce3

Reply:
removed gio